scan/scan.go

```diff
@@ -30,16 +30,25 @@ func Scan(dialog zenity.ProgressDialog) {
 		func() checks.Check {
 			return checks.WindowsDefender(registrymock.LOCAL_MACHINE, registrymock.LOCAL_MACHINE)
 		},
-		checks.LastPasswordChange,
 		func() checks.Check {
-			return checks.LoginMethod(registrymock.LOCAL_MACHINE)
```

Comment:
Waarom is LoginMethod terug naar zonder arguments?

Reply:
Blijkbaar pulled die dit niet van nieuwe main, heb het nu gefixed